cmd/ctl/pkg/install/install.go

```diff
@@ -140,6 +135,7 @@ func NewCmdInstall(ctx context.Context, ioStreams genericclioptions.IOStreams) *
 	cmd.Flags().StringVar(&options.ChartName, "chart-name", "cert-manager", "Name of the chart to install")
 	cmd.Flags().MarkHidden("chart-name")
 	cmd.Flags().BoolVar(&options.DryRun, "dry-run", false, "Simulate install and output manifest")
+	cmd.Flags().BoolVar(&options.printManifests, "print-manifests", false, "Print manifests to stdout")
```

Comment:
What is the difference between `DryRun` and `printManifests`?

Reply:
Sure, but here we also print the manifests when the dry-run option is set: https://github.com/cert-manager/cert-manager/blob/35cccef21f302e4e88d35a1cbb2668bfd805ac1b/cmd/ctl/pkg/install/install.go#L113-L116
So, right now `--dry-run --print-manifests` will print the manifests twice.
I think we can just remove the `--print-manifests` flag.